README.md

```diff
@@ -102,12 +112,23 @@ spec:
 | `LEGO_KUBE_API_URL` | n | `http://127.0.0.1:8080` | API server URL |
 | `LEGO_LOG_LEVEL` | n | `info` | Set log level (`debug|info|warn|error`) |
 
-
 ## Full deployment examples
 
 - [Nginx Ingress Controller](examples/nginx/)
 - [GCE Load Balancers](examples/gce/)
 
+## Troubleshooting
+
+When interacting with *kube-lego*, its a good idea to run with `LEGO_LOG_LEVEL=debug` for more verbose details.
+Additionally, be aware of the automatically created resources (see environment variables) when cleaning up or testing.
+
+Possible resources for *help*:
+
+* `#coreos` on freenode
```

Comment:
> Either we create a chat location for kube-lego

Who will be in that channel to provide support?  You would likely need to raise such an idea with the core developers and get them on board to do that.

> We get explicit permission from other places to post their channel here.
> We rename as "k8s chat channels" instead of "places for help"

This doesn't solve the fact that those groups are a general support for those getting started with K8s, not this project.  Selling them as supporting this project when they don't.  If a user needs help with k8s basics, they can look for that on their own, not inside something listed under this projects banner.

> We do absolutely nothing
Is absolutely the best choise - there is no support.
The main page of this project reads `Non-production use case 😆` under `Requirements`.  While it doesn't directly say "unsupported", that's pretty much what it means in my reading.  Any user who choses to run this in production does so at their own risk and under their own volition.


I'll note that the `Authors` secion as well lists that this was done under jetstack.io, a container consultancy.  In such case, the best case of getting support is likely to hire them.